lib/ui/screen-components/home/drawer/drawer-header/index.tsx

```
@@ -1,22 +1,46 @@
-import { View, Text } from "react-native";
-import { useTranslation } from "react-i18next";
+import { View, Text, Image } from 'react-native'
```

ESLint

[error] 'Image' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)